spec/predicates/human-review-crev.md

```diff
@@ -0,0 +1,161 @@
+# Predicate type: Dependency Reviews (crev)
+
+Type URI: (tentative) https://in-toto.io/attestation/human-review/crev/v0.1
+
+Note: At the time of writing, the crev project uses a separate versioning
+scheme, with the current version being -1. In future, this predicate may adopt
+a URI associated with that project.
+
+Version: v0.1.0
+
+## Purpose
+
+This attestation type is used to describe the results of human review of
+dependency source code. The format is based on the
+[crev project](https://github.com/crev-dev/crev).
+
+## Use Cases
+
+As noted above, crev enables social review of popular open source software
+dependencies. A crev review includes information such as the thoroughness of
+the review, understanding of the source code, and a final rating.
+
+## Model
+
+Most modern software have external dependencies. Dependency review is the
+process of reviewing and verifying the source code of a particular
+dependency, and can be performed by one or more of several actors in the supply
+chain. The developer importing a new dependency can perform the review or a
+dedicated security team can be tasked with it.
+
+## Schema
```

Comment:
To my more general comment about which Crev spec is authoritative, I suggest indicating here which format this predicate follows. As indicated on the main project repo, I'm assuming the spec specified in [cargo-crev](https://github.com/crev-dev/cargo-crev/blob/master/crev-lib/rc/doc/editing-code-review.md) is the authoritative one.